dcompiler.dd

```diff
@@ -45,7 +49,7 @@ $(H2 $(LNAME2 requirements, Requirements and Downloads))
     $(LI Xcode)
     )
     $(FREEBSD
-    $(LI 32 bit x86 FreeBSD 7.1 operating system)
+    $(LI 32 bit x86 and 64 bit x86-64 FreeBSD operating system (11 or later))
```

Comment:
https://github.com/braddr/d-tester/issues/70#issuecomment-368166348

Reply:
You can put comments in the .dd file by putting them inside $(COMMENT my illuminating explanation) which expands to nothing.

Reply:
Oh that comment was just intended as a help for the reviewers, s.t. it's clear why I made this change.